models.txt

```diff
@@ -1,3 +1,6 @@
+icatcher-realistic
```

Comment:
move this to line 12 (before this change) where is at end of our sdxl community models.